ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -460,17 +460,6 @@ export const LeonardoAppLauncher = fp.flow(
       return appendNotebookFileSuffix(this.getNotebookName());
     }
 
-    private async initializeNotebookCookies() {
-      return await this.runtimeRetry(() =>
```

Comment:
Should we keep the retry functionality? I think an error modal telling the user to refresh on the first failure may be sufficient.